<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>466</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>便利贴时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="stickyTime"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>便利贴内容：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txtNoteContext"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>删除的编号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spItem">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOK">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDelete">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="mainWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>AddNote()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelete</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>DeleteNote()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>DeleteNote()</slot>
  <slot>AddNote()</slot>
 </slots>
</ui>
